api-tests/core/content-type-builder/components.test.api.js

```diff
@@ -68,6 +68,7 @@ describe('Content Type Builder - Components', () => {
           component: {
             category: 'default',
             displayName: 'Some Component',
+            icon: 'Calendar',
```

Comment:
Why did you opt for uppercase component icons? Just out of curiosity ...

Reply:
I personally don't mind the uppercase if it can avoid maintaining a map or having to do string manipulation

Reply:
^^^ this, you're create a mental link between the exports of the DS & the DB – could be extremely dangerous, imo create a concrete list of icons you want to be used.

Reply:
That looks much better to me, thanks! Now the question is: does this still need to be uppercase, or should this use the icon keys from the mapping :)